### DISCUSSION TITLE
ESP32-WROVER replacement

### DISCUSSION THREAD

**Discussion by comejar** [ctag-fh-kiel/ctag-straempler#25] [Q&A]
It seems the one on the bom isn't available on mouser anymore

https://www.mouser.co.uk/ProductDetail/356-ESP32WVIE23264UC

would this be a suitable replacement? looks to be the exact same thing just a later released version hence difference naming

**Comment by ctag-fh-kiel** [comment]
Hi, I recommend taking the one with 16MB Flash ESP32-WROVER-IE-N16R8, it should have this Mouser id: 356-ESP32WRVIE2864UC

**Reply by comejar** [reply]
Thankyou! I think thats the one I originally saw when i compared specs and saw the flash was the same as the bom one but I had that many tabs open I ended up linking the wrong one on here anyway haha
